apps/src/components/map-layers/map-legend-inner-s2d.tsx

```diff
@@ -87,99 +50,20 @@ interface ProbabilityStatementProps {
  */
 interface ProbabilityStatementTooltipRow {
 	/**
-	 * Percentile threshold value (0-100)
+	 * Description Term item title
 	 *
-	 * @example 80 for "80th percentile"
-	 */
-	value: number;
-
-	/**
-	 * User-facing category label
-	 *
-	 * @example 'Unusually high', 'Above normal'
+	 * @example 'Unusually high'
 	 */
 	label: string;
 
 	/**
-	 * Template string with `%s` placeholder for percentile value
+	 * Description Term Details -- one or many lines under that label
 	 *
-	 * @example 'Above the %s percentile' becomes "Above the 80th percentile"
+	 * @example 'Above the 66th percentile ...'
 	 */
 	text: string;
-
-	/**
-	 * Explanatory clarification in parentheses
-	 *
-	 * @example 'top fifth of historical data', 'middle three-fifths'
-	 */
-	parens: string;
 }
```

Comment:
This type is now basically an unit for [Description Term and one or more Description Data](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/dt) element (`dl > dt dd`) where `label` is what goes in `dt`, and text in `dd`. Here it's only one.